Source/WebCore/platform/graphics/filters/skia/FEColorMatrixSkiaApplier.cpp

```diff
@@ -92,13 +94,12 @@ bool FEColorMatrixSkiaApplier::apply(const Filter&, const FilterImageVector& inp
         return false;
     }
 
-    SkPaint paint;
-    paint.setColorFilter(SkColorFilters::Matrix(matrix.data()));
-
-    auto* canvas = result.beginRecording();
-    canvas->drawImage(nativeImage->platformImage(), 0, 0, { }, &paint);
-    result.finishRecording();
+    static_cast<GraphicsContextSkia*>(&resultImage->context())->drawCustom([=](SkCanvas& canvas) {
```

Comment:
Use`&` instead `=` for lambda capture, since it's sync call, we don't want to copy any value.

Reply:
No lambdas anymore!